lnst/Tests/RDMABandwidth.py

```diff
@@ -0,0 +1,85 @@
+from abc import ABC, abstractmethod
+import logging
+
+from lnst.Tests.BaseTestModule import BaseTestModule
+from lnst.Common.Parameters import BoolParam, IntParam, IpParam, ListParam, StrParam
+from lnst.Common.ExecCmd import exec_cmd
+from lnst.Common.Utils import is_installed
+
+
+class RDMABandwidthBase(ABC, BaseTestModule):
+    """
+    RDMA / InfiniBand bandwidth test using `ib_send_bw`
+    """
+    device_name = StrParam(mandatory=True)
+    duration = IntParam()
+    port = IntParam()
+    cpu_bind = ListParam(type=IntParam())
+    size = IntParam(default=65536)
+
+    def run(self) -> bool:
+        self._res_data = {}
+        if not is_installed("ib_send_bw"):
+            self._res_data["msg"] = "perftest is not installed on this machine!"
```

Comment:
this mentions `perftest` not installed?

Reply:
`perftest` is the package that provides `ib_send_bw` program and some others. See [linux-rdma/perftest](https://github.com/linux-rdma/perftest). Should I change to message to something like `"perftest package (provides ib_send_bw) ..."`?

Reply:
ah, i see, makes sense, sure, please change the message, that should be a bit more helpful.